test/core/RandomVariables.jl

```diff
@@ -1,9 +1,9 @@
 using Turing, Random
 using Turing: Selector, reconstruct, invlink, CACHERESET, SampleFromPrior
 using Turing.RandomVariables
-using Turing.RandomVariables: uid, cuid, getvals, getidcs,
+using Turing.RandomVariables: uid, cuid, getidcs,
```

Comment:
Are we aware of the test coverage of the `TypedVarInfo`?

Reply:
I made sure the test coverage of `TypedVarInfo` is no less than that of `UntypedVarInfo`. But I will see if I can improve it.